src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/NullCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.References
+{
+    class NullCommand : IProjectSystemUpdateReferenceOperation
```

Comment:
We generally add explicit access modifiers.

```suggestion
    internal class NullCommand : IProjectSystemUpdateReferenceOperation
```